### ISSUE TITLE
TV Shows Library Incorrect Image Banners

### ISSUE THREAD

**Issue by Artiume** [jellyfin/jellyfin-androidtv#310]
On the Home Page, the tv show pictures are a mismatch between some being the newer Primary Cards while others are using the old Thumb images. Haven't tested pulling new metadata yet. Just noticed that it's quite the mismatch haha

**Comment by nielsvanvelzen** [comment]
I prefer the primary images and think they should be used everywhere. Backdrop should be used as a backdrop, not a thumbnail.

**Comment by Artiume** [comment]
I completely agree. From the looks of it, we just need to change the logic for individual episodes. They pull the banner instead of the primary image.
